portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,39 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Date;
+
+import java.util.ArrayList;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    
+    String jsonStr = convertToJsonUsingGson();
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(jsonStr);
   }
+
+  
+
+  private String convertToJsonUsingGson() {
+    ArrayList<String> messages = new ArrayList<String>();
+    /*messages.add("\"startTime\": " +  "\"" + serverStats.getStartTime() + "\"");
+    messages.add("\"currentTime\": " + "\"" + serverStats.getCurrentTime() + "\"";);
+    */
+    messages.add("hello, it's me");
+    messages.add("i was wondering it after all");
+    messages.add("these years");
+    Gson gson = new Gson();
+    String json = gson.toJson(messages);
+    return json;
```

Comment:
Same comment here, is option 1 easier to read than option 2?

Option 1:
```java
  String json = gson.toJson(messages);
  return json;
```

Option 2:
```java
  return gson.toJson(messages);
```